src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -182,3 +183,25 @@
  :wallet/clear-last-collectible-details
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet] dissoc :last-collectible-details)}))
+
+(rf/reg-event-fx :wallet/trigger-share-collectible
+ (fn [_ [{:keys [title uri]}]]
+ {:fx [[:effects.share/open
+          {:title   title
+           :message title
+           :url     uri}]]}))
+
+(rf/reg-event-fx :wallet/share-collectible
```

Comment:
This is WIP as need to figure out how to dispatch effects later :/

Reply:
We can create them, I think it's just a wrapper on `js/setTimeout`, could be called `:fx-later`.
Another option might be creating `:wallet/trigger-share-collectible` as an effect instead of as an event and it could use `js/setTimeout` internally.

btw, why do we want to dispatch an effect later?

Reply:
> btw, why do we want to dispatch an effect later?

I believe the idea is to wait for the bottom sheet to close and trigger the system share sheet, as the user pressed the `Share Collectible` option and didn't want to see the sheet still open after the operation.

Reply:
yep exactly